### ISSUE TITLE
RBO subscribes to mask images with wrong size

### ISSUE THREAD

**Issue by pazeshun** [start-jsk/jsk_apc#1564]
When I was running `setup_softkinetic.launch` and `main.launch`, I sometimes got the message like following:
```
/home/baxter/ros/indigo/src/start-jsk/jsk_apc/jsk_apc2016_common/python/jsk_apc2016_common/rbo_segmentation/probabilistic_segmentation.py:568: VisibleDeprecationWarning: boolean index did not match indexed array along dimension 1; dimension is 601 but corresponding boolean dimension is 319
  self.likelihood_images[feature_name][object_name][self.has3D_mask] = Utils.backproject(self.likelihoods[feature_name][object_name], self.feature_images[feature_name][self.has3D_mask])
```

**Comment by knorth55** [comment]
it's warning not failing, right?
can you get topic when you get the warning?

**Comment by knorth55** [comment]
can you check the topic is proper segmented if you get topic?

**Comment by yuyu2172** [comment]
@pazeshun 
Thanks for reporting.
This problem probably occurs because RBO node and mask node are not synchronized.

**Comment by knorth55** [comment]
put more precise title

**Comment by pazeshun** [comment]
In fact, I don't know much about this problem. I didn't feel any bad effect about this. But when I told this to @yuyu2172 , he said this expressed failing of RBO. So I decide to open this issue.
I want @yuyu2172 to change title and put more information.

**Comment by knorth55** [comment]
@yuyu2172 fixed via #1592 ?

**Comment by yuyu2172** [comment]
Yes. I am closing this issue.
